react-responsive-modal/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-responsive-modal",
+  "name": "@huksley/react-responsive-modal",
```

Comment:
Can you revert this please?

Reply:
omg, sorry I am using my package now. removed.